src/components/Footer.tsx

```diff
@@ -320,7 +320,7 @@ const Footer = ({ lastDeployLocaleTimestamp }: FooterProps) => {
         <Button
           variant="outline"
           isSecondary
-          onClick={() => scrollIntoView("__next")}
+          onClick={() => scrollIntoView("div.mx-auto")}
```

Comment:
Ah, I see that `h1` by default doesn't really position things well.. I'm just going to switch this to use `body` here instead then to avoid any weirdness with this selector in the future.

```suggestion
          onClick={() => scrollIntoView("body")}
```